### PR TITLE
Lana fixes for lan-party 2018

### PR DESCRIPTION
**Pull request by mcbloch**

### CODE REVIEW

**Comment by ninewise** [comment]
You seem to have accidently pushed your `.idea` directory 😮 

Kindly fix this by

1. reading `man git-ignore` (you might also want to check `man git-config` to set a global gitignore)
2. reading (the description and --index-filter options of) `man git-filter-branch`
3. understanding and executing following snippet:
   ```
   git filter-branch --index-filter 'git rm -r --cached --ignore-unmatch .idea' master..updates
   git push --force-with-lease origin updates
   ```

**Comment by mcbloch** [comment]
Yep, my bad. Read and executed, idea is gone :)

File: README.md [hunk 1]
```diff
@@ -1,20 +1,58 @@
...
+    bundle exec rails db:migrate RAILS_ENV=test
+    
+    # execute
+    bundle exec rake
+
+## Programma structuur
+
+### Data flow
+
+Het programmma is gebouwt rond redux. Elke client kan acties sturen (gedefinieerd in [actions](app/assets/javascripts/actions), gepubliceert door [publishedAction](app/assets/javascripts/actions/utils)) welke naar de server gestuurd worden,
```
Comment (iasoon) [inline README.md:55]: gebouwd

**Review by iasoon — APPROVED** [review]
Sorry for taking so long to review this, I totally forgot about it. The changes look good to me! It's lovely that this old project is getting some new attention, thank you for your efforts @maximebloch ! Rock on!

**Comment by mcbloch** [comment]
@Robbe7730 merge?

**Comment by wschella** [comment]
I'll merge, I'll also give you full access rights to the repo.

**Pull request merged**